spec/requests/user_pages_spec.rb

```diff
@@ -16,7 +16,7 @@
     describe "pagination" do
 
       before(:all) { 20.times { FactoryGirl.create(:user)}}
-      after(:all) { User.delete_all}
+      # after(:all) { User.delete_all}
```

Comment:
Please explain why?

Reply:
As we are droping and creating a new database on each test run so it is not necessary.